src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -0,0 +1,226 @@
+import { ChainId, Currency } from '@uniswap/sdk'
```

Comment:
what is the change here?

Reply:
![image](https://user-images.githubusercontent.com/2352112/109647013-2223a800-7b59-11eb-9ee0-0b525a0b749b.png)

I followed the procedure. The mod compares line by line, easy to see the change as I show in the image above. 

Also, this refers to the change in the description:

> * Adapted the confirmation modal 

Additionally, the index.tsx documents why it was created: 
![image](https://user-images.githubusercontent.com/2352112/109647325-7169d880-7b59-11eb-9a4e-07e6bcb5ab56.png)